init-database.sh

```diff
@@ -0,0 +1,17 @@
+# From project root
+cd ./migrations/ || exit
+
+initDatabaseScript=$(basename -- "init_database.sql")
+echo "Start init database from $initDatabaseScript script"
+# listOfDoneMigrationScript=("./history/*")
+
+ errorlog=$(mktemp)
+    trap 'rm -f "$errorlog"' EXIT
+    pwcheck="$(psql postgresql://postgres:a!11111111@localhost:5432/social_network -v ON_ERROR_STOP=1 -f "$initDatabaseScript" 0 < "$errorlog")"
```

Comment:
Db url should come from env vars. If there are no required env vars then exit the script